packages/component-library/src/HorizontalBarChart/HorizontalBarChart.js

```diff
@@ -189,7 +207,7 @@ const HorizontalBarChart = ({
               animate
             />
           )}
-          {!stacked && (
+          {!stacked && !grouped && (
```

Comment:
This can be wrapped within the condition above unless you're trying to keep them separate

Reply:
I think was keeping separate because that's how they were before -- there's a lot of chart clean up for after Demo Day